src/cryptography/x509/extensions.py

```diff
@@ -936,6 +937,13 @@ def __init__(self, oid, critical, value):
     critical = utils.read_only_property("_critical")
     value = utils.read_only_property("_value")
 
+    def _validate(self):
+        # Validate the extension if it has a _validate method
+        try:
+            self.value._validate()
+        except AttributeError:
```

Comment:
I don't like this, it silently consumes an `AttributeError` throw by `_validate()`